Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
 
+    void SetGeometry(GeometryTGeo *g) { fGeometry=g; }
```

Comment:
yes, I'm fully aware of the situation and I also know that this can only improved gradually ;-) I know it's a long way, so lets start going early ...